### ISSUE TITLE
When blur is applied with compton, the bar background does not update on wallpaper change

### ISSUE THREAD

**Issue by Qwerty-Space** [polybar/polybar#2145]
## Describe the issue
My bar is translucent, and when a blur effect is applied with compton, and the wallpaper changes polybar does not always update

**Expected behavior:**

For polybar to display correctly

**Actual behavior:**

**Was it working before?**
This does not occur when the blur is disabled

## To Reproduce
*A minimal but complete config with which the problem occurs:*

[polybar conf](https://github.com/Qwerty-Space/dotfiles/blob/85e34429637535e3d20d9478eb19d04e580d6b48/polybar/.config/polybar/config#L14)

[compton conf](https://github.com/Qwerty-Space/dotfiles/blob/85e34429637535e3d20d9478eb19d04e580d6b48/compton/.config/compton/compton-kawase.conf#L35)

*List any other steps needed to reproduce the issue besides starting polybar with the config you posted above.*

## Polybar Log
*Post everything polybar outputs to the terminal when you run it and the issue occurs below*
Nothing relevant in the logs

## Screenshots
*If applicable, add screenshots to help explain your problem.*
![2020-07-13_17-22](https://user-images.githubusercontent.com/6967592/87330847-0d929100-c531-11ea-845f-0f3ade00b6f7.png)
![2020-07-13_17-53](https://user-images.githubusercontent.com/6967592/87331418-d375bf00-c531-11ea-9347-a1130a86e395.png)

## Environment:
* WM: Herbstluftwm
* Compositor: compton-tryone-git
* Distro: Arch
* Output of `polybar -vvv`:
```
polybar 3.4.3

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /usr/bin/c++
Compiler flags: -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now 
```

## Additional context
*Add any other context that you think is necessary about the problem here.*

**Comment by patrick96** [comment]
I can't reproduce this at all with the exact same setup. What kind of graphics driver do you have?

**Comment by Qwerty-Space** [comment]
It only seems to occur sometimes.  I'm using `xf86-video-amdgpu`

**Comment by frebib** [comment]
`compton-tryone-git` is old. Try installing `picom-git` instead. It contains the dual-kawase blur functionality and contains a bunch of other new improvements too

**Comment by Qwerty-Space** [comment]
I've been meaning to move over soon, it's just a matter of finding the time

**Comment by patrick96** [comment]
@Qwerty-Space Were you able to try out picom and see whether that issue occurs there?

**Comment by Qwerty-Space** [comment]
@patrick96 no, picom didn't have kawase blur at the time. I'm not sure if it has it in the repos yet. There's several reasons beyond polybar why I need to update, but I'm just holding out for kawase blur haha 

I will update when I can
